modules/spring-mock-mvc/src/test/java/io/restassured/module/mockmvc/ParserTest.java

```diff
@@ -60,7 +60,7 @@
         then().
                 parser("some/thing", Parser.JSON).
                 statusCode(200).
-                contentType(equalTo("some/thing")).
+                contentType(equalTo("some/thing;charset=ISO-8859-1")).
```

Comment:
Is this due to changes in Spring?

Reply:
@johanhaleby 
Thank you for your reply.
Yes, org.springframework:spring-web 4.3.19 adds default 'charset'.